aeron-cluster/src/main/java/io/aeron/cluster/ConsensusModuleAgent.java

```diff
@@ -754,6 +755,10 @@ void onCatchupPosition(
                     logRecordingId, logPosition, Long.MAX_VALUE, channel, ctx.logStreamId()));
                 follower.catchupReplayCorrelationId(archive.lastCorrelationId());
             }
+            else if (null != follower)
+            {
+                System.out.println("Existing catchup already running: " + follower);
```

Comment:
Do we really want a `System.out` here? Could be this a `ClusterEvent` instead?

Reply:
No we don't, I'll remove that.